src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -192,6 +192,10 @@ trait WorkflowComponent {
       sqlu"update WORKFLOW set status = ${newStatus.toString}, status_last_changed = ${new Timestamp(System.currentTimeMillis())}, record_version = record_version + 1 where status = ${currentStatus.toString}"
     }
 
+    def abortWorkflowsOfStatus(submissionId: UUID, currentStatus: WorkflowStatus): WriteAction[Int] = {
```

Comment:
I would make this more generic and have it take both the current status as well as the desired status, similar to batchUpdateStatus above except specific to a submission